zebra-rpc/src/methods/get_block_template_rpcs.rs

```diff
@@ -316,7 +315,7 @@ where
     fn get_block_template(
```

Comment:
Can you remove the part of the TODO about HexData, because this PR does that?

Reply:
Yep, removed [here](https://github.com/ZcashFoundation/zebra/pull/5870/commits/8420b75013c1a05ea11a9799d1f87bd338c3c27c#diff-c0bec6265a8d68255f80da0e5f499035c5c0181cc218c3d68805094e2c7815f5L314).